packages/cli-plugin-scaffold-react-package/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@webiny/cli-plugin-scaffold-react-package",
+  "version": "4.4.0",
+  "description": "Generate a React package similar to our 'app-' packages",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/webiny/webiny-js.git",
+    "directory": "packages/cli-scaffold-graphql-service"
+  },
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/webiny/webiny-js/issues"
+  },
+  "homepage": "https://github.com/webiny/webiny-js#readme",
+  "dependencies": {
+    "execa": "^4.0.3",
+    "find-up": "^4.1.0",
+    "lodash.kebabcase": "^4.1.1",
+    "ncp": "^2.0.0"
+  },
+  "devDependencies": {
+    "@shelf/jest-mongodb": "^1.1.5"
```

Comment:
This dependency is not needed as the actual scaffolding package will never have tests with mongo.